api/envoy/extensions/http/stateful_session/cookie/v3/cookie.proto

```diff
@@ -0,0 +1,52 @@
+syntax = "proto3";
+
+package envoy.extensions.http.stateful_session.cookie.v3;
+
+import "google/protobuf/duration.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.http.stateful_session.cookie.v3";
+option java_outer_classname = "CookieProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Cookie based stateful session extension]
+
+// This extension allows the session state to be tracked via cookies.
```

Comment:
I was thinking a bit about multiple hops. For example, client -> external LB (envoy) -> internal LB (envoy) -> app.

I am assuming this would be broken if both envoy's configure this, right? Since both Envoys would have the cookie set but they would be configured the value to the host's of `app` or `internal LB`.

Is that right?

I don't think there is any code changes here, just something users should be aware of. If we have this we can set a different cookie at each hop I think

Reply:
Good point. Thanks for calling it out. I will add some new comments as note. 😄